themes/app/templates/Includes/Footer.ss

```diff
@@ -1 +1,47 @@
-<footer>Irsan Arisandy @ SilverStripe &copy; $Now.Year</footer>
+<footer>
+    <div class="container footer__partners">
+        <div class="row">
+            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6 ">
```

Comment:
I think all you'll need here is `col-lg-2 col-md-3 col-sm-6`, give that a go and see if it still works as you expect

Reply:
If it does work then update the other classes too :)

Reply:
@pjayme taking out the `col-6` seems to not break the content into two columns when the screen is below the small break point.